docs/source/user.rst

```diff
@@ -0,0 +1,254 @@
+  metheglin.
+
+* A recipe with apple juice *and* spices will be classified as an open
+  category mead.
+
+.. note:: These classifications are just recommendations.  If they are
+          incorrect or incomplete, please submit :doc:`bug reports
+          <bugfeature>` as appropriate.
+
+Recipes also have titles and descriptions for organizational purposes
+as well.
+
+Recipes can be created from the recipe select page as well as from
+existing batches by pressing the 'Create recipe from batch' button on
+the batch's change page.
+
+Batches
+~~~~~~~
+
+Batches are actual instantiations of recipes.  Sometimes brew day
+doesn't go exactly as planned, so ingredient substitutions and other
+changes are possible.  Additionally, samples are associated with
+batches.
+
+Like recipes, batches have titles and descriptions.  Unlike recipes,
+batches also have names.  Batch names are composed of unique strings
+representing brewing events and a batch letter indicating the sequence
+in which the batch was brewed.  The author uses batch names like "SIP
+26 C", where "SIP 26" represents the brewing event and "C" indicates
+that this was the third batch brewed during that event.
+
+.. note:: Any non-empty string can be used as an event identifier.
+
+Batches can be created from the recipe's change page by pressing the
+'Create batch from recipe' button.  Placeholder values for the
+batch-specific information must be changed immediately to avoid
+conflict with other batches.
+
+.. todo:: There is an open issue about this problem.
+
+Products
+~~~~~~~~
+
+Products are batches which have been bottled.  Once the bottling is
+complete, labels can be generated based on the product's ingredients
+and other characteristics.  Jars of products that have been labeled
+can be entered into the inventory management system.
+
+Product names, titles, and descriptions are all inherited from their
+batches.  Product titles and descriptions are currently used for both
+labels and distribution.
+
+When a batch is bottled into jars, the Jars field of the batch's
+change page must be updated and the batch saved.  At this point labels
+can be printed and applied to the jars.  A product can then be created
+from that batch by pressing the 'Create product from batch` button on
+the batch's change page.
+
+Samples
+~~~~~~~
+
+Samples are taken throughout the lifetime of the batch.  The following
+information is collected for each sample:
+
+* Temperature (in degrees Fahrenheit)
+* Specific gravity
+* Notes (traditionally used for sensory data)
+
+These samples are used to calculate the percent alcohol of the mead,
+and are convenient for storing information which can be referenced
+when making labels.
+
+Storage
+-------
+
+Once the mead is produced, it must be stored in such a manner as to
+facilitate easy retrieval of specific jars.  This requires an
+inventory management system.  The largest unit of storage in the IMS
```

Comment:
Since for the rest of the time you call it an IMS, I would do this in the first sentence about it: "This requires an inventory management system (IMS)."